signer/src/api/info.rs

```diff
@@ -0,0 +1,505 @@
+//! Handler for the `/info` endpoint.
+
+use axum::{extract::State, response::IntoResponse, Json};
+use serde::Serialize;
+
+use crate::{
+    bitcoin::BitcoinInteract,
+    config::Settings,
+    context::Context,
+    stacks::api::StacksInteract,
+    storage::{
+        model::{BitcoinBlockHash, StacksBlockHash},
+        DbRead,
+    },
+};
+
+use super::ApiState;
+
+#[derive(Debug, Serialize)]
+pub struct InfoResponse {
+    pub bitcoin: BitcoinInfo,
+    pub stacks: StacksInfo,
+    pub dkg: DkgInfo,
+    pub build_info: BuildInfo,
+    pub timestamp: String,
+}
+
+#[derive(Debug, Serialize)]
+pub struct BuildInfo {
+    pub rust_version: String,
+    pub git_revision: String,
+    pub target_arch: String,
+    pub target_env_abi: Option<String>,
```

Comment:
You can use `&'static str` for all of these, it should work just fine.

Reply:
[807b6ba](https://github.com/stacks-network/sbtc/pull/1251/commits/807b6ba69aee8fd1c8f5651870d2177c4056da51)